src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
It's not a simple find and replace, e.g. this needs to be completely rewritten 
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/statistics_controller.rb#L201

But anyway, closed for now...